README.dev.md

```diff
@@ -5,7 +5,7 @@ Steps for releasing:
 3. Review `CHANGELOG.md` for completeness and correctness. Update its release date.
```

Comment:
This whole doc needs to be updated.  See https://github.com/maxmind/minfraud-api-node/blob/main/README.dev.md